base_vat_optional_vies/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================================
+Optional validation of VAT via VIES
+===================================
+
+This module extends base_vat module features allowing to know if VIES
+validation was passed or not.
```

Comment:
You should mention in any place that the validation is optional, unlike standard one, that blocks the write.